src/firefly/html/firefly.html

```diff
@@ -20,7 +20,8 @@
                     ],
                 options : {
                     MenuItemKeys: {maskOverlay:true},
-                    catalogSpacialOp: 'polygonWhenPlotExist'
+                    catalogSpacialOp: 'polygonWhenPlotExist',
+                    charts: {chartEngine: 'plotly'}
```

Comment:
Yes, I put it in everywhere so that they will all pick up the option and we can work on it.  IrsaViewer in the IFE repository does not have this option s set.